### DISCUSSION TITLE
Tasklists unlocked!

### DISCUSSION THREAD

**Discussion by Lamparter** [RiversideValley/.github#1] [Show and tell]
![tasklist-hero](https://user-images.githubusercontent.com/71598437/217174876-ec4f2863-5baa-442d-a17e-c7774e1c804d.png)

### 🎉 Tasklists!

These will allow us @RiversideValley to integrate part of our [projects](https://github.com/orgs/RiversideValley/projects) into our issues as a preview - allowing the creation of a tracker[^1] for a wide feature!

cc. @NoobNotFound @developerx-official @onein528 

[^1]: Like a more advanced version of [FluentHub's tracker](https://github.com/fluenthub/fluenthub/blob/main/.github/ISSUE_TEMPLATE/tracker.yml)

**Comment by Lamparter** [comment]
@NoobNotFound should I integrate these into @RiversideValley/emerald's issue templates?

**Reply by NoobNotFound** [reply]
why not,yeah!

**Reply by Lamparter** [reply]
Done. See https://github.com/RiversideValley/.github/commit/4ad61887dcbc912440b54d58d580bc5ff71b2591 and https://github.com/RiversideValley/Emerald/commit/8af9df9140015d5dab355d32502cc28db348303c
